classes/user/DAO.php

```diff
@@ -154,13 +154,14 @@ public function getIds(Collector $query): Collection
      */
     public function getByUsername(string $username, bool $allowDisabled = false): ?User
     {
-        $row = DB::table($this->table)
-            ->where('username', '=', $username)
+        $row = DB::table('users')
+            ->whereRaw("lower(username) = lower('$username')")
```

Comment:
Don't forget to add bindings! https://github.com/pkp/pkp-lib/pull/8227#discussion_r975563608